src/Adapter/Driver/Mysqli/Result.php

```diff
@@ -51,9 +51,9 @@ class Result implements
     protected $nextComplete = false;
 
     /**
-     * @var bool
+     * @var mixed
      */
-    protected $currentData = false;
+    protected $currentData = null;
```

Comment:
This is a BC break for anything inheriting from this class. Sadly, the class is not `final`, so these changes land in future releases (`develop`)

Reply:
Yes, I'm planning it for zend-db 3.0.0. I'll point to `develop`, thanks!

Reply:
I'm going to release this in 2.10.0 since it's a rollback of 2.8 behaviour.

Reply:
Thanks! We are still on our local patched version of 2.9, so it's going to be great to be able to switch back to the main repo.